dev_setup.sh

```diff
@@ -41,6 +41,9 @@
 # exit on any error
 set -Ee
 
+TOP=$( cd $( dirname $0 ) && pwd -L )
+cd $(dirname $0)
```

Comment:
We can swap these to make it less redundant:
```bash
cd $(dirname $0)
TOP=$(pwd -L)
```

Reply:
Done!